Dangerfile

```diff
@@ -9,6 +9,26 @@
   end
 end
 
+# Ensure a clean commits history
 if commits.any? { |c| c.message =~ /^Merge branch '#{branch_for_merge}'/ }
   fail("Please rebase to get rid of the merge commits in this PR")
 end
+
+# Request a CHANGELOG entry, and give an example
+has_app_changes = !modified_files.grep(/lib/).empty?
+if !modified_files.include?("CHANGELOG.md") && has_app_changes
+  fail ("Please include a CHANGELOG entry to credit yourself! \nYou can find it at [CHANGELOG.md](https://github.com/CocoaPods/CocoaPods/blob/master/CHANGELOG.md).", sticky: false)
+
+  pr = env.request_source.pr_json
+  markdown <<-MARKDOWN
+Here's an example of your CHANGELOG entry:
+
+```
+* #{pr.title}
```

Comment:
maybe add a `#{'  '}` at the end of this line, so if they copy/paste they get the two spaces?